aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java

```diff
@@ -515,4 +513,26 @@ private StringBuilder parseForeignKeys(SQLiteTable table) {
         }
         return builder;
     }
+
+    // Utility method to create SQL for Primary key on table
+    private StringBuilder createPrimaryKey(@NonNull ModelSchema modelSchema) {
+        final StringBuilder builder = new StringBuilder();
+        final List<String> indexFields = modelSchema.getPrimaryIndexFields();
+        if (indexFields.size() > 0) {
+            builder.append("PRIMARY KEY")
+                    .append(SqlKeyword.DELIMITER)
+                    .append("(");
+            int numIndexFieldsAdded = 0;
+            for (String filed : indexFields) {
+                numIndexFieldsAdded++;
+                builder.append(SqlKeyword.DELIMITER)
+                        .append(filed);
+                if (numIndexFieldsAdded != indexFields.size()) {
+                    builder.append(",");
+                }
+            }
+            builder .append(")");
```

Comment:
```suggestion
            builder.append(")");
```

Reply:
I assume this was to line up with the `append()`s above the for loop?  Not sure how style works on this team, but this looks better to my eyes 😅